specs/interop/token-bridging.md

```diff
@@ -72,26 +101,200 @@ contract OptimismSuperchainERC20 is ERC20, ISuperchainERC20 {
 }
 ```
 
-## L1 Native Tokens
+[Disclaimer]
+
+- Threat the following code as experimental. It will have errors and will change function names. The goal is to discuss functionalities.
+- Current function names are open to discussion.
+
+## Implementation
+
+### Cross Chain transfers
+
+The standard will include the following three external functions:
+
+- `bridgeERC20(uint256 amount, uint256 chainId)`
+- `bridgeERC20To(address to, uint256 amount, uint256 chainId)`
+- `finalizeBridgeERC20(address to, uint256 amount)`
+
+The first two functions will burn `amount` tokens and initialize a message to the `L2ToL2CrossChainMessenger` to mint the `amount` in the destination (`to` address in `chainId`).
+
+`finalizeBridgeERC20(address to, uint256 amount)` will process incoming messages from the `L2ToL2CrossChainMessenger` initiated from the same token on a different `chainId` and mint `amount` to the `to` address.
+
+A possible implementation would look like this:
+
+```solidity
+function bridgeERC20(uint256 _amount, uint256 _chainId) external returns (bool success) {
+    success = bridgeERC20To(msg.sender, _amount, _chainId);
+  }
+
+function bridgeERC20To(address _to, uint256 _amount, uint256 _chainId) public returns (bool) {
```

Comment:
Thoughts on replacing the `bridgeXXX` with `sendXXX` as the goal is to make it feel like one chain, so we can make the API more like the messenger which doesn't use the bridge language, per https://github.com/ethereum-optimism/specs/pull/71#discussion_r1631562710